### PR TITLE
Style links in guide documents

### PR DESCRIPTION
**Pull request by ben181231**
connect #425 

P.S. This PR would regard links in guide document are external links, except the following host name:
- localhost
- docs.skygear.io
- docs-staging.skygear.io

### CODE REVIEW

File: utils/markdown-link-attributer.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-param-reassign */
+
+// Ref. https://github.com/crookedneighbor/markdown-it-link-attributes/blob/7e83cca/index.js
```
Comment (rickmak) [inline utils/markdown-link-attributer.js]: I assume this code is not the same with `npm install markdown-it-link-attributes`, so you copy here.

If it is copies, I would just put our options here. The `markdown-loader.js` is getting complex.

Comment (ben181231) [inline utils/markdown-link-attributer.js]: Yes. This snippet is inspired by `markdown-it-link-attributes`. 

Since we have some specific rules for the anchor tag rendering. I try to make it general and put it here. For the specific rules, I put it on `markdown-loader.js` so that this snippet can be reused later.

I can see the `markdown-loader.js` is getting complex. Would it be better to extract those rules to another file?

Comment (rickmak) [inline utils/markdown-link-attributer.js]: Okay, as long as we are well aware of the issue. Let do it in a separate PR.
